orchestra/todos/serializers.py

```diff
@@ -39,12 +43,16 @@ class Meta:
             'completed',
             'skipped_datetime',
             'start_by_datetime',
-            'due_datetime')
+            'due_datetime',
+            'activity_log')
         read_only_fields = ('id',)
 
 
 class TodoWithQASerializer(serializers.ModelSerializer):
```

Comment:
Do we want to log activity on the TodoWithQA? Or alternatively, do we want to log activity on TodoWithoutQA?:)

Reply:
`TodoWithoutQA` or `TodoWithQA` is just a way to make sure that QA data is only visible to the assigned worker.

Reply:
I'm saying we shouldn't log activity not attributable the assigned worker. For example, in the current design would we end up logging a reviewer's expand/collapse activity?

Reply:
Adding worker id to the activity_log.